src/plugins/console/public/lib/utils/utils.test.js

```diff
@@ -193,4 +193,64 @@ describe('Utils class', () => {
 
     expect(utils.getResponseWithMostSevereStatusCode(undefined)).toBe(undefined);
   });
+
+  describe('replaceVariables', () => {
+    const variablesMock = [
+      { name: 'v1', value: 'test' },
+      { name: 'v2', value: 'true' },
+      { name: 'v3', value: '{"f": "test"}' },
+      { name: 'v4', value: '{"v1": "${v1}", "v6": "${v6}"}' },
+      { name: 'v5', value: '[{"t": "test"}]' },
+      { name: 'v6', value: '1' },
+    ];
+
+    function testVariables(data, variables, expected) {
+      const result = utils.replaceVariables(data, variables);
+      expect(result).toEqual(expected);
+    }
+
+    it('should replace variables in url and body', () => {
```

Comment:
I think we can simplify these tests and improve readability by removing `variablesMock`, inlining the mock values within the tests, and removing the need to wrap the `testVariables` arguments in arrays. WDYT?

```js
function testVariables(data, variables, expected) {
  const result = utils.replaceVariables([data], [variables]);
  expect(result).toEqual([expected]);
}

it('should replace variables in url and body', () => {
  testVariables(
    {
      url: '${var}/search',
      data: ['{\n  "f": "${var}"\n}']
    },
    { name: 'var', value: 'test' },
    {
      url: 'test/search',
      data: ['{\n  "f": "test"\n}'],
    },
  );
});
```